java/core/src/main/java/co/worklytics/psoxy/ConfigRulesModule.java

```diff
@@ -3,35 +3,30 @@
 import co.worklytics.psoxy.gateway.ConfigService;
 import co.worklytics.psoxy.gateway.ProxyConfigProperty;
 import co.worklytics.psoxy.rules.PrebuiltSanitizerRules;
+import co.worklytics.psoxy.rules.RuleSet;
 import co.worklytics.psoxy.rules.RulesUtils;
 import dagger.Module;
 import dagger.Provides;
-import org.apache.commons.lang3.ObjectUtils;
 
-import javax.swing.text.html.Option;
 import java.util.Optional;
 import java.util.function.BiFunction;
-import java.util.function.Function;
-import java.util.function.Supplier;
 import java.util.logging.Logger;
 
 @Module
 public class ConfigRulesModule {
 
-    public static final String PATH_TO_RULES_FILES = "/rules.yaml";
 
     @Provides
-    static Rules rules(Logger log, RulesUtils rulesUtils, ConfigService config) {
+    static RuleSet rules(Logger log, RulesUtils rulesUtils, ConfigService config) {
 
-        BiFunction<Optional<Rules>, String, Optional<Rules>> loadAndLog = (o, msg) -> {
+        BiFunction<Optional<RuleSet>, String, Optional<RuleSet>> loadAndLog = (o, msg) -> {
             if (o.isPresent()) {
                 log.info(msg);
             }
             return o;
         };
 
-        return loadAndLog.apply(rulesUtils.getRulesFromFileSystem(PATH_TO_RULES_FILES), "Rules: loaded from file system")
```

Comment:
dropped support for explicit `rules.yaml` file. in practice, I think that's no easier than using env variable OR AWS SystemParameter (and those are quite easy, as can change rules without re-deploy)